whirlpool/auth.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import annotations
```

Comment:
why is this needed for?

Reply:
Typing

Reply:
I removed it and got no errors. What typing do you refer to that requires this?

Reply:
Any time typing is used say  `-> int | None` you need the annotations import or it will complain.
Maybe we removed the one I had that needed it.

Reply:
https://dev.to/tiangolo/the-future-of-fastapi-and-pydantic-is-bright-3pbm

Reply:
I don't think it is needed anymore, unless you are using an old python version. Since HA uses 3.10 as minimum, we should also do the same here. Maybe your python env was still using a previous version?

Reply:
According to the link, it didn't make it into 3.10.